README.md

```diff
@@ -29,102 +39,26 @@ This project has 2 parts: the Docker environment and a set of tools for theme de
 ```
 # copy the files
 git clone https://github.com/visiblevc/wordpress-starter.git
-rm -rf .git Dockerfile run.sh README.md CHANGELOG.md ISSUE_TEMPLATE.md
+
+# navigate to example directory
+cd wordpress-starter/example
 
 # start the website at localhost:8080
-docker-compose up
+docker-compose up -d && docker-compose logs -f wordpress
```

Comment:
Yeah, the benefit to doing my way is that you get only the wordpress logs, as you stated, and also you can just <kbd>ctrl</kbd>-<kbd>c</kbd> after it's done building without it tearing down the application.

You can detach from a forground process by hitting <kbd>ctrl</kbd>-<kbd>p</kbd> followed by <kbd>ctrl</kbd>-<kbd>q</kbd> as well (I think), but that's too difficult to remember.